File: 3794-reverse-string-prefix/3794-reverse-string-prefix.cpp
class Solution {
public:
    string reversePrefix(string s, int k) {
        string str;
        for(int i=k-1;i>=0;i--){
            str.push_back(s[i]);
        }
        for(int j=k;j<s.size();j++){
            str.push_back(s[j]);
        }
        return str;
    }
};

File: 3794-reverse-string-prefix/3794-reverse-string-prefix.java
class Solution {
    public String reversePrefix(String s, int k) {
        StringBuilder str=new StringBuilder();
        for(int i=k-1;i>=0;i--){
           str.append(s.charAt(i));
        }

        for(int i=k;i<s.length();i++){
            str.append(s.charAt(i));
        }
        return str.toString();
    }
}